examples/step-2/doc/results.dox

```diff
@@ -1,14 +1,43 @@
 <h1>Results</h1>
 
-The program has, after having been run, produced two sparsity
-patterns. We can visualize them by opening the <code>.svg</code> files in a web browser.
-
-The results then look like this (every point denotes an entry which
-might be nonzero; of course the fact whether the entry actually is
+The program has, after having been run, produced two files of
+DoF locations and sparsity patterns each (once for the original numbering
+and once after renumbering), along with one mesh file.
+
+Let us start with the DoF locations. There is no particularly convenient
+program to visualize this kind of information, but we can resort to
+<a href="http://www.gnuplot.info/">GNUPLOT</a> (one of the simpler visualization
+programs; maybe not the easiest to use since it is command line driven, but
+also universally available on all Linux and other Unix-like systems). The
+command that produces the following pictures reads as follows:
+```
+plot [-0.1:2.1][-1.1:1.1] "mesh.gnuplot" with lines, "dof-locations-1.gnuplot" using 1:2:3 with labels point offset .3,.2 font "4,6"
```

Comment:
@tjhei This is where the command to create the picture is. The documentation of the function that creates the data for these figures also links to this location.

Reply:
Excellent. How do add a ``set size ratio -1`` or whatever it is called?

Reply:
I don't know. What do you want to do? I don't know that command...

Reply:
so that the inside of the mesh is shown with the correct aspect ratio (as a circle).

Reply:
Ah, nice! I didn't know about that command. I've updated the figures on the website now.